### PR TITLE
Feat/alphabetized types

### PR DESCRIPTION
**Pull request by polomani**
I made a script which sorts declarations in file alphabetically. (this feature is not supported by prettier yet, I didn't find any other solution)
And then ran it for types.ts.
This is needed to be able to check for interfaces changes easily.
And also makes easier to avoid conflicts.

### CODE REVIEW

File: scripts/alphabetizeTypes.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+const comments = new Comments();
+const parser = new TypescriptParser();
+
+const FILENAME = argv.i;
+const HEADER = argv.h;
+
+(async () => {
+    const original = readFileSync(FILENAME).toString();
+    const parsedCode = await parser.parseFile(FILENAME).catch(err => {
+        console.error(err);
```
Comment (ghost) [inline scripts/alphabetizeTypes.js:32]: Could you wrap the entire function with a catch instead?

**Review by ghost — COMMENTED** [review]
This is very cool!

**Comment by ghost** [comment]
`yargs` doesn't support node9 and are failing the build.

File: scripts/alphabetizeTypes.js (hunk 1 above)
Comment (polomani) [reply scripts/alphabetizeTypes.js:32]: yes

**Review by ghost — APPROVED** [review]

**Pull request merged**